types/chrome/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "dependencies": {
+    "@puppeteer/replay": "^1.0.0"
```

Comment:
I'm not a fan of adding [1MB-plus-dependency](https://packagephobia.com/result?p=%40puppeteer%2Freplay) to a type package. I think this should be dropped. I think that if someone needs those types they can install it independently, even if this d.ts explicitly mentions `import type {Step, UserFlow} from '@puppeteer/replay'`

If that's not acceptable, how about venturing replay’s d.ts instead? https://unpkg.com/browse/@puppeteer/replay@2.9.0/lib/main.d.ts